src/Jaeger.Core/Reporters/RemoteReporter.cs

```diff
@@ -239,7 +239,7 @@ public RemoteReporter Build()
                 }
                 if (_sender == null)
                 {
-                    _sender = new UdpSender();
+                    _sender = NoopSender.Instance;
```

Comment:
I would rather throw exception, null should not be valid, and Noop as default doesn't make much sense.

Reply:
We also have an `NoopReporter` and a `NoopMetricsFactory`, so it would be natural to also include an `NoopSender`. But from a user-perspective, when a non-`NoopReporter` is chosen.

The only use case I can currently think of that this allows for is getting metrics without the need for a sender.

Reply:
BTW: Java is also using the NoopSender in this case. I will add a warning as Java is doing:
https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-core/src/main/java/io/jaegertracing/internal/senders/SenderResolver.java#L60